src/Microsoft.Azure.NotificationHubs/RegistrationDescription.cs

```diff
@@ -217,6 +215,15 @@ internal string FormattedETag
 
         internal abstract RegistrationDescription Clone();
 
+        /// <summary>
+        /// Returns platform-specific Psn handle.
```

Comment:
Can you change "Psn" to "Pns". The acronym is short for _Platform Notification System_:

> Push notifications are delivered through platform-specific infrastructures called Platform Notification Systems (PNSes).

(from https://docs.microsoft.com/en-us/azure/notification-hubs/notification-hubs-push-notification-overview#how-push-notifications-work)

Reply:
My bad,
Had PSN (PlayStation Network) in mind when adding the property :)